src/Sarif/GitInformation.cs

```diff
@@ -0,0 +1,180 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading;
+
+namespace Microsoft.CodeAnalysis.Sarif
```

Comment:
>Sarif [](start = 33, length = 5)

I did not put this in a `Git` subdirectory/namespace, since the other three files (IConsoleCapture.cs, ConsoleStreamCapture.cs, and ExternalProcess) are non Git-specific, so this would have been the only type in the namespace.